changelogs/unreleased/4407-move-cfcache-files-to-project-root.yml

```diff
@@ -0,0 +1,8 @@
+---
+description: The compiler cache (.cfc) files are now stored in the .cfcache directory in the root of the inmanta project instead of in the __cfcache__ directory in the inmanta modules.
+issue-nr: 4407
+issue-repo: inmanta-core
+change-type: patch
+destination-branches: [master, iso5, iso4]
```

Comment:
```suggestion
destination-branches: [iso5, iso4]
```
iso4 should be removed as well if you're creating a separate PR there.

Reply:
Indeed. I completely forgot about that.

Reply:
It's a very common mistake so perhaps we should consider removing the need by having merge tool check on which branches the change entry exists already.

Reply:
Could be difficult considering other PRs might already be in flight.

Reply:
It's worth investigating in my opinion.

Reply:
I'll create a ticket

Reply:
inmanta/irt#1199